.github/workflows/integration.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       matrix:
         python-version: [3.7]
-        subset: ['upload_datatype', 'extended_metadata', 'kubernetes', 'not (upload_datatype or extended_metadata or kubernetes)']
+        subset: ['upload_datatype_irods']
```

Comment:
Please revert the changes to this file

Reply:
Sure

Reply:
Can you restore the tests please ?

Reply:
Hmm, somehow I missed this. Pushing it next.